bundles/net.tourbook.statistics/src/net/tourbook/statistics/graphs/StatisticTooltipUI_CategorizedData.java

```diff
@@ -803,8 +800,8 @@ private void updateUI() {
       _lblDistance                           .setText(distance                      == 0 ? UI.EMPTY_STRING : FormatManager.formatDistance_Summary      (distance / 1000.0));
       _lblDistance_Unit                      .setText(distance                      == 0 ? UI.EMPTY_STRING : UI.UNIT_LABEL_DISTANCE);
 
-      _lblElevationUp                        .setText(elevationUp_WithMeasurement   == 0 ? UI.EMPTY_STRING : FormatManager.formatElevation_Summary     (elevationUp_WithMeasurement));
-      _lblElevationUp_Unit                   .setText(elevationUp_WithMeasurement   == 0 ? UI.EMPTY_STRING : UI.UNIT_LABEL_ELEVATION);
+      _lblElevationUp                        .setText(elevationUp_Summary   == 0 ? UI.EMPTY_STRING : FormatManager.formatElevation_Summary     (elevationUp_Summary));
+      _lblElevationUp_Unit                   .setText(elevationUp_Summary   == 0 ? UI.EMPTY_STRING : UI.UNIT_LABEL_ELEVATION);
```

Comment:
`elevationUp_Summary` was wrong at this location, changed to `elevationUp`

Reply:
Bad copy/paste from my part

Reply:
Fixed